etsd/core/forms.py

```diff
@@ -3,12 +3,13 @@
 from django.utils.translation import ugettext_lazy as _
 from authorities.models import Authority
 from dal import autocomplete
+from django.utils.translation import ugettext as _
 
 
 class AuthorityUsersModelForm(forms.ModelForm):
     #users = forms.ModelMultipleChoiceField(widget=forms.CheckboxSelectMultiple, queryset=get_user_model().objects.all(), label=_('Select users'), required=False, )
     users = forms.ModelMultipleChoiceField(
-        widget=autocomplete.ModelSelect2Multiple(url='user-autocomplete',), 
+        widget=autocomplete.ModelSelect2Multiple(url='/users/user-autocomplete',), 
```

Comment:
Please don't pass the url of the view, use the name instead. See here how it's done: https://django-autocomplete-light.readthedocs.io/en/3.5.1/tutorial.html#register-the-autocomplete-view

Reply:
I.e pass the `user-autocomplete` name here.